spec/duration.html

```diff
@@ -1346,220 +1323,161 @@ <h1>
 
     <emu-clause id="sec-temporal-add24hourdaystonormalizedtimeduration" type="abstract operation">
       <h1>
-        Add24HourDaysToNormalizedTimeDuration (
-          _d_: a Normalized Time Duration Record,
+        Add24HourDaysToTimeDuration (
+          _d_: a time duration,
           _days_: an integer,
-        ): either a normal completion containing a Normalized Time Duration Record or a throw completion
+        ): either a normal completion containing a time duration or a throw completion
       </h1>
       <dl class="header">
         <dt>description</dt>
         <dd>
-          It returns a normalized time duration that is the sum of _d_ and the number of 24-hour days indicated by _days_, throwing an exception if the result is greater than the maximum normalized time duration.
+          It returns a time duration that is the sum of _d_ and the number of 24-hour days indicated by _days_, throwing an exception if the result is greater than the maximum time duration.
           This operation should not be used when adding days relative to a Temporal.ZonedDateTime, since the days may not be 24 hours long.
         </dd>
       </dl>
       <emu-alg>
-        1. Let _result_ be _d_.[[TotalNanoseconds]] + _days_ × nsPerDay.
+        1. Let _result_ be _d_ + _days_ × nsPerDay.
         1. If abs(_result_) > maxTimeDuration, throw a *RangeError* exception.
-        1. Return Normalized Time Duration Record { [[TotalNanoseconds]]: _result_ }.
+        1. Return _result_.
       </emu-alg>
     </emu-clause>
 
-    <emu-clause id="sec-temporal-addnormalizedtimedurationtoepochnanoseconds" type="abstract operation">
+    <emu-clause id="sec-temporal-addtimedurationtoepochnanoseconds" type="abstract operation">
       <h1>
-        AddNormalizedTimeDurationToEpochNanoseconds (
-          _d_: a Normalized Time Duration Record,
+        AddTimeDurationToEpochNanoseconds (
+          _d_: a time duration,
           _epochNs_: a BigInt,
         ): a BigInt
       </h1>
       <dl class="header">
         <dt>description</dt>
         <dd>
-          It adds a normalized time duration _d_ to an exact time in nanoseconds since the epoch, _epochNs_, and returns a new exact time.
+          It adds a time duration _d_ to an exact time in nanoseconds since the epoch, _epochNs_, and returns a new exact time.
           The returned exact time is not required to be valid according to IsValidEpochNanoseconds.
         </dd>
       </dl>
       <emu-alg>
-        1. Return _epochNs_ + ℤ(_d_.[[TotalNanoseconds]]).
+        1. Return _epochNs_ + ℤ(_d_).
       </emu-alg>
     </emu-clause>
 
-    <emu-clause id="sec-temporal-comparenormalizedtimeduration" type="abstract operation">
+    <emu-clause id="sec-temporal-comparetimeduration" type="abstract operation">
       <h1>
-        CompareNormalizedTimeDuration (
-          _one_: a Normalized Time Duration Record,
-          _two_: a Normalized Time Duration Record,
+        CompareTimeDuration (
```

Comment:
I considered inlining AOs like this one, but I think implementations will probably represent time durations as a struct, not a single number. Personally, I'd do it as a 64-bit number of seconds and 32-bit number of subseconds. Given that, I think it still makes sense for operations like these to be considered "methods" of time durations.